<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>EditServiceDialog</class>
    <widget class="QDialog" name="EditServiceDialog">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>457</width>
                <height>126</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>Edit Service</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
            <item>
                <layout class="QFormLayout" name="formLayout">
                    <property name="fieldGrowthPolicy">
                        <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
                    </property>
                    <item row="0" column="0">
                        <widget class="QLabel" name="label">
                            <property name="text">
                                <string>Service Name</string>
                            </property>
                            <property name="buddy">
                                <cstring>serviceName</cstring>
                            </property>
                        </widget>
                    </item>
                    <item row="0" column="1">
                        <widget class="QLineEdit" name="serviceName">
                            <property name="toolTip">
                                <string>The label associated with this service.</string>
                            </property>
                        </widget>
                    </item>
                    <item row="1" column="0">
                        <widget class="QLabel" name="label_2">
                            <property name="text">
                                <string>Service Address:</string>
                            </property>
                            <property name="buddy">
                                <cstring>serviceAddress</cstring>
                            </property>
                        </widget>
                    </item>
                    <item row="1" column="1">
                        <widget class="QValidatedLineEdit" name="serviceAddress">
                            <property name="toolTip">
                                <string>The address associated with this service.</string>
                            </property>
                        </widget>
                    </item>
                    <item row="2" column="0">
                        <widget class="QLabel" name="label_3">
                            <property name="text">
                                <string>Service Type:</string>
                            </property>
                            <property name="buddy">
                                <cstring>serviceType</cstring>
                            </property>
                        </widget>
                    </item>
                    <item row="2" column="1">
                        <widget class="QComboBox" name="serviceType">
                            <property name="toolTip">
                                <string>The type of service.</string>
                            </property>
                        </widget>
                    </item>
                </layout>
            </item>
            <item>
                <widget class="QDialogButtonBox" name="buttonBox">
                    <property name="orientation">
                        <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="standardButtons">
                        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
                    </property>
                </widget>
            </item>
        </layout>
    </widget>
    <customwidgets>
        <customwidget>
            <class>QValidatedLineEdit</class>
            <extends>QLineEdit</extends>
            <header>qvalidatedlineedit.h</header>
        </customwidget>
    </customwidgets>
    <resources/>
    <connections>
        <connection>
            <sender>buttonBox</sender>
            <signal>accepted()</signal>
            <receiver>EditServiceDialog</receiver>
            <slot>accept()</slot>
            <hints>
                <hint type="sourcelabel">
                    <x>20</x>
                    <y>20</y>
                </hint>
                <hint type="destinationlabel">
                    <x>20</x>
                    <y>20</y>
                </hint>
            </hints>
        </connection>
        <connection>
            <sender>buttonBox</sender>
            <signal>rejected()</signal>
            <receiver>EditServiceDialog</receiver>
            <slot>reject()</slot>
            <hints>
                <hint type="sourcelabel">
                    <x>20</x>
                    <y>20</y>
                </hint>
                <hint type="destinationlabel">
                    <x>20</x>
                    <y>20</y>
                </hint>
            </hints>
        </connection>
    </connections>
</ui>
